misk-policy/src/main/kotlin/misk/policy/opa/RealOpaPolicyEngine.kt

```diff
@@ -56,6 +56,31 @@ class RealOpaPolicyEngine @Inject constructor(
     return parseResponse(document, returnType, response)
   }
 
+  /**
+   * Evaluate / Query a document with given input of raw JSON.
+   * This will connect to OPA via a retrofit interface and perform a /v1/data/{document} POST.
+   * This consumes raw JSON for corner cases where developers need to do queries the automatic
+   * JSON serialization doesn't support.
+   *
+   * @param document Name or Path of the OPA document to query.
+   * @param input Input data to be supplied to OPA at evaluation time. Must be valid JSON.
+   * @param returnType Return shape to be JSONified from OPA.
+   * @throws PolicyEngineException if the request to OPA failed or the response shape didn't match R.
+   * @throws IllegalArgumentException if no document path was specified.
+   * @return Response shape R from OPA.
+   */
+  override fun <R: OpaResponse> evaluateRawJsonInput(
```

Comment:
super nit: since this function overrides/implements the `OpaPolicyEngine` interface, you don't have to copy paste the javadoc here.

Reply:
Agreed. Since the existing methods do that, I followed convention.